packages/usa-input-mask/src/content/usa-input-mask-custom-text.json

```diff
@@ -0,0 +1,12 @@
+{
+  "customInvalidAlphaText": "Ingrese un carácter de letra aquí",
+  "customInvalidNumericText": "Ingrese un carácter de número aquí",
```

Comment:
Note that the Spanish might need tweaking. I was just trying to apply a realistic use case.